src/module_3/train.py

```diff
@@ -0,0 +1,98 @@
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.base import BaseEstimator
+from sklearn.metrics import precision_recall_curve, auc, roc_auc_score
+import statsmodels.api as sm
+from statsmodels.discrete.discrete_model import LogitResults
+import pandas as pd
+import numpy as np
+import datetime
+import joblib
+
+from functions import three_way_split_time, get_best_treshold, pick_features_for_splits, logit_significance
```

Comment:
What is `functions`? Did we miss a file in the PR? 
I don't see much here: https://pypi.org/project/functions/

Reply:
![imagen](https://github.com/user-attachments/assets/9424a4c4-af19-4451-a5d3-f907a6a7ac30)

It's a file I used to write down a few functions. I just send the path to it in the PR